src/components/ui/frame.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import * as React from "react"
+import Image from "next/image";
+import type { ImgHTMLAttributes } from "react";
+import {
+    FrameUI,
+    fallbackFrameContext,
+} from "@frames.js/render";
+import { signFrameAction, FarcasterSigner } from '@frames.js/render/farcaster'
+import { useFrame } from "@frames.js/render/use-frame";
+import {AccountObjectType} from "@/stores/useAccountStore";
+
+export function FrameImageNext(
+    props: ImgHTMLAttributes<HTMLImageElement> & { src: string }
+): React.JSX.Element {
+    return (
+        <Image
+            {...props}
+            alt={props.alt ?? ""}
+            sizes="100vw"
+            height={0}
+            width={0}
+        />
+    );
+}
+
+
+export default function Frame( { homeFrameUrl, account } : { homeFrameUrl: string, account?: AccountObjectType }) {
+    const farcasterSigner: FarcasterSigner | undefined = account ? {
+        fid: account?.id,
+        status: account?.status,
+        publicKey: account?.publicKey,
+        privateKey: account?.privateKey,
+    } : undefined;
+
+    const frameState = useFrame({
+        homeframeUrl: homeFrameUrl,
+        connectedAddress: account?.publicKey,
```

Comment:
connectedAddres should be picked from wagmi / viewm `useAccount`. The account.publicKey relates to the signer on the farcaster protocol https://docs.farcaster.xyz/learn/architecture/overview

Reply:
oh yeah, thanks, will read Farcaster docs soon enough!